TTS/utils/io.py

```diff
@@ -37,23 +40,37 @@ def copy_model_files(config, out_path, new_fields):
     copy_config_path = os.path.join(out_path, "config.json")
     # add extra information fields
     config.update(new_fields, allow_new=True)
-    config.save_json(copy_config_path)
```

Comment:
if you don't get lazy you can add save_fsspec to https://github.com/coqui-ai/coqpit :) 
Otherwise, it is better to use 

```python
config.to_json()
# or
config.to_dict()
```

Reply:
Oh but I am lazy :D

Jokes aside, I will for sure make a PR to allow Coqpit to save with fsspec once this is in, but I figured adding this small bit here for now has the advantage of not depending on a dependency update so it's easier to test etc. For now I switched to `config.to_dict()`